app/views/instrument_price_policies/_table.html.haml

```diff
@@ -0,0 +1,58 @@
+- clazz = price_policies.first.class
```

Comment:
Yeah, it's really just to save some typing with the uses of `human_attribute_name` below.

I could go more with what I have in the next file, where there's `price_policy = price_policies.first` and then a bunch of `price_policy.class.human_attribute_name`. Either way, it would probably be worth being consistent between the two file.